File: app/src/main/java/com/coppel/rhconecta/dev/business/models/CoppelGeneralParameterResponse.java
package com.coppel.rhconecta.dev.business.models;

import java.io.Serializable;

public class CoppelGeneralParameterResponse implements Serializable{

    private Meta meta;

    public Meta getMeta() {
        return meta;
    }

    public void setMeta(Meta meta) {
        this.meta = meta;
    }

    public class Meta {

        private String status;

        public String getStatus() {
            return status;
        }

        public void setStatus(String status) {
            this.status = status;
        }
    }
}

File: app/src/main/java/com/coppel/rhconecta/dev/data/home/model/get_main_information/CoppelGeneralParameterResponse.kt
package com.coppel.rhconecta.dev.data.home.model.get_main_information

import java.io.Serializable

open class CoppelGeneralParameterResponse: Serializable {
    private var meta: Meta? = null

    fun getMeta(): Meta? {
        return meta
    }

    fun setMeta(meta: Meta?) {
        this.meta = meta
    }

    class Meta : Serializable {
        var status: String? = null
    }
}